debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/InternalDebugRevenueCatScreen.kt

```diff
@@ -10,39 +12,66 @@ import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.collectAsState
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.lifecycle.viewmodel.compose.viewModel
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.PurchasesTransactionException
 import com.revenuecat.purchases.debugview.models.InternalDebugRevenueCatScreenViewModel
+import com.revenuecat.purchases.debugview.models.InternalDebugRevenueCatScreenViewModelFactory
 import com.revenuecat.purchases.debugview.models.SettingGroupState
 import com.revenuecat.purchases.debugview.models.SettingScreenState
 import com.revenuecat.purchases.debugview.models.SettingState
 import com.revenuecat.purchases.debugview.settings.SettingGroup
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.StoreTransaction
+import com.revenuecat.purchases.models.SubscriptionOption
 import kotlinx.coroutines.flow.MutableStateFlow
 
 @Composable
 internal fun InternalDebugRevenueCatScreen(
-    screenViewModel: DebugRevenueCatViewModel = viewModel<InternalDebugRevenueCatScreenViewModel>(),
+    onPurchaseCompleted: (StoreTransaction) -> Unit,
+    onPurchaseErrored: (PurchasesTransactionException) -> Unit,
+    screenViewModel: DebugRevenueCatViewModel = viewModel<InternalDebugRevenueCatScreenViewModel>(
+        factory = InternalDebugRevenueCatScreenViewModelFactory(onPurchaseCompleted, onPurchaseErrored),
+    ),
 ) {
     Column(
         modifier = Modifier
             .verticalScroll(rememberScrollState())
             .fillMaxWidth()
             .padding(bottom = 16.dp),
     ) {
+        val state = screenViewModel.state.collectAsState().value
+        DisplayToastMessageIfNeeded(screenViewModel, state = state)
         Text(
             text = "RevenueCat Debug Menu",
             style = MaterialTheme.typography.h5,
             modifier = Modifier.padding(horizontal = 16.dp, vertical = 16.dp),
         )
-        screenViewModel.state.collectAsState().value.toGroupStates().forEach { SettingGroup(it) }
+        state.toGroupStates().forEach { SettingGroup(it) }
     }
 }
 
+@Composable
+private fun DisplayToastMessageIfNeeded(viewModel: DebugRevenueCatViewModel, state: SettingScreenState) {
+    state.toastMessage?.let { toastMessage ->
+        Toast.makeText(
+            LocalContext.current,
+            toastMessage,
+            Toast.LENGTH_LONG,
+        ).show()
```

Comment:
I'm showing the toast as part of the debug view. I think it's ok, but I was wondering whether it would be better to just leave to the developer to handle it as they see fit, since there is a callback. Thoughts?